src/acl/Gadgets.cc

```diff
@@ -49,19 +49,16 @@ aclGetDenyInfoPage(AclDenyInfoList ** head, const char *name, int redirect_allow
     debugs(28, 8, HERE << "got called for " << name);
 
     for (A = *head; A; A = A->next) {
-        AclNameList *L = NULL;
-
         if (!redirect_allowed && strchr(A->err_page_name, ':') ) {
             debugs(28, 8, HERE << "Skip '" << A->err_page_name << "' 30x redirects not allowed as response here.");
             continue;
         }
 
-        for (L = A->acl_list; L; L = L->next) {
-            if (!strcmp(name, L->name)) {
-                debugs(28, 8, HERE << "match on " << name);
+        for (const auto L : A->acl_list) {
```

Comment:
Please avoid SBuf copying if possible. We should also avoid using capitalized names for local variables.

```suggestion
        for (const auto &aclName: A->acl_list) {
```